draft-ietf-tls-esni.md

```diff
@@ -112,68 +112,51 @@ comes from {{RFC8446}}; Section 3.
+|    |                   | pub.example.com  |    | priv.example.com |
+|    |                   | (2001:DB8::1111) |    | (2001:DB8::EEEE) |
+|    |                   |                  |    |                  |
+|    |<---  TLS 1.3  --------------------------->|                  |
+|    |                   |                  |    |                  |
+|    |<-  ESNI Trans-  ->|                  |    |                  |
+|    |     formation     |                  |    |                  |
++----+                   +------------------+    +------------------+
+Client                   Client-Facing Server       Backend Server
 ~~~~
-{: #shared-mode title="Shared Mode Topology"}
+{: #topology title="Topology"}
 
-In Shared Mode, the provider is the origin server for all the domains
-whose DNS records point to it and clients form a TLS connection directly
-to that provider, which has access to the plaintext of the connection.
+The client sends a transformed ClientHello on the wire, that contains an
+encrypted server name (see {{esni-transformation}}). The client-facing server
+acts as a proxy between the client and the backend server, decrypting the server
+name sent by the client, and forwards the restored ClientHello as well as the
+following TLS records to the backend server, which is identified during the
+inverse transformation. The client-facing server is also responsible for
+providing the up-to-date ESNI keys should there be a mismatch (see
+{{server-behavior}}).
 
-~~~~
-                +--------------------+       +---------------------+
-                |                    |       |                     |
-                |   2001:DB8::1111   |       |   2001:DB8::EEEE    |
-Client <------------------------------------>|                     |
-                | public.example.com |       | private.example.com |
-                |                    |       |                     |
-                +--------------------+       +---------------------+
-                  Client-Facing Server            Backend Server
-~~~~
-{: #split-mode title="Split Mode Topology"}
-
-In Split Mode, the provider is *not* the origin server for private
-domains. Rather the DNS records for private domains point to the provider,
-but the provider's server just relays the connection back to the
-backend server, which is the true origin server. The provider does
-not have access to the plaintext of the connection. In principle,
-the provider might not be the origin for any domains, but as
-a practical matter, it is probably the origin for a large set of
-innocuous domains, but is also providing protection for some private
-domains. Note that the backend server can be an unmodified TLS 1.3
-server.
+The backend server can be a stock TLS 1.3 server.
 
+The client-facing server and the backend server can be collocated. When they are
+not, the communication between these two servers MUST be encrypted (e.g., by
```

Comment:
collocated -> colocated. Both are technically correct, but "colocate" is the modern usage in the computer industry.

Reply:
Fixed in 92e3116.